### DISCUSSION TITLE
【keyball61】ファームウェア書き込みには成功するが認識しない

### DISCUSSION THREAD

**Discussion by rikirikrik** [Yowkees/keyball#485] [Q&A] [answered]
表題の件について、
左手側のファームウェアの書き込み後、OLEDモジュールの表示を確認すると、添付画像のような状態となっており、キーボード入力も反応しません。
基盤のハンダについても問題無いように見えますが、原因がわからない状況です。
ご助力いただけますと幸いです。

■書き込んだファームウェア
https://remap-keys.app/catalog/RZSU1CrvEW4lns0ww5BM/firmware
Keyball61_test_v1.2.0

■確認したこと
・右手側は正常に動作していたので、ProMicroとOLEDを取り外し、左手側に付け替えて、再度ファームウェアの書き込みを行ったが、OLEDの状態は変わらず、キーボード入力も反応しない
・反対に動作しなかったProMicroとOLEDを、動作していた右手に付け替えたところ、右手側では動作した。（なので左手基盤の問題？）

![スクリーンショット 2024-02-17 122805](https://github.com/Yowkees/keyball/assets/86538387/a4c93655-d52f-4e8c-aa4e-ec3d05043aed)
![IMG_8695](https://github.com/Yowkees/keyball/assets/86538387/1bd0008f-ec98-4819-8bdb-1eb0a2a91884)
![IMG_8696](https://github.com/Yowkees/keyball/assets/86538387/1fdc0bdf-6474-4c7c-bcd0-0d8146d87ed6)
![IMG_8697](https://github.com/Yowkees/keyball/assets/86538387/2c2ba489-0601-408a-add6-9a3321a3ea41)

**Comment by Ryo624** [comment, marked as answer]
ご連絡ありがとうございます。

添付しました写真の赤い丸のDATA、GNDと書かれた箇所の端子のはんだが隣の端子に触れているように見えますので、隣の端子に触れないようにしてみてください。
また、緑色の丸で示した箇所が端子にはんだが十分馴染んでいないように見えますので、はんだ付けをやり直してみてください。
![305595325-2c2ba489-0601-408a-add6-9a3321a3ea41](https://github.com/Yowkees/keyball/assets/143698349/a9914c54-591b-42be-ab86-c76656abb2d8)

**Reply by rikirikrik** [reply]
ご確認いただきありがとうございます。
ご指摘いただいた個所のハンダし直しを行ったところ、正常に認識することが確認できました。
本当にありがとうございます！

**Reply by Ryo624** [reply]
無事に解決していただきましてありがとうございます！

ぜひ末永くご愛用いただければと思います。
